devito/interfaces.py

```diff
@@ -4,11 +4,12 @@
 from tools import aligned
 import os
 from signal import signal, SIGABRT, SIGINT, SIGSEGV, SIGTERM
+from tempfile import gettempdir
 import sys
 import atexit
 
 
-__all__ = ['DenseData', 'TimeData', 'PointData']
+__all__ = ['DenseData', 'TimeData', 'PointData', 'MemmapManager', 'SymbolicData']
```

Comment:
Fine, just remember that the docs are actually intended for **user-facing API**. It's nice that you are cleaning up the developer-specific things too, but that's not as important as the public API. That being said, maybe we should start to think about which parts we actually want users to see? For example, do we really want to expose the `function_manager` or `cgen_wrapper` modules to the wider public?